packages/splitio-adapter/modules/adapter/adapter.js

```diff
@@ -50,13 +53,14 @@ const ensureUser = user => ({
   ...user,
 });
 
-const initializeUserContext = (clientSideId, user) => {
+const initializeClient = (authorizationKey, user) => {
```

Comment:
This notion of user is very interesting to me. I always felt each adapter/provider of flags will have a user notion and would provide custom properties to be fed in for targeting. For splitio, according to their [docs](https://docs.split.io/docs/javascript-sdk-overview) you only pass in a key.

Reply:
Yes absolutely!